sotopia/install/cli.py

```diff
@@ -0,0 +1,203 @@
+            if system == "Darwin":
+                console.log(
+                    """Docker is not installed.
+                    Please check https://docs.docker.com/desktop/install/mac-install/,
+                    or install it using homebrew with `brew install --cask docker`.
+                    And then run this command again.
+                    """
+                )
+            elif system == "Linux":
+                console.log(
+                    """Docker is not installed.
+                    Please check https://docs.docker.com/engine/install/ubuntu/.
+                    And then run this command again.
+                    """
+                )
+            else:
+                console.log(
+                    """Docker is not installed.
+                    Please check https://docs.docker.com/desktop/install/windows-install/.
+                    And then run this command again.
+                    """
+                )
+        else:
+            if system == "Windows":
+                console.log("""For Windows, unfortunately only docker is supported.
+                    Check the official documentation:
+                    https://redis.io/docs/latest/operate/oss_and_stack/install/install-stack/windows/.
+                """)
+                exit(1)
+            elif system == "Darwin":
+                # check if homebrew is installed
+                try:
+                    subprocess.check_output("command -v brew", shell=True)
+                except subprocess.CalledProcessError:
+                    console.log(
+                        """Homebrew is required for install redis without docker on MacOS.
+                        Please check https://brew.sh/.
+                        And then run this command again.
+                        """
+                    )
+                    exit(1)
+
+    if load_database is None:
+        load_database = (
+            Prompt.ask(
+                "Do you want to load the database with initial sotopia(-pi) data?",
+                choices=["Yes", "No"],
+                default="Yes",
+                console=console,
+            )
+            == "Yes"
+        )
+
+    if load_database and custom_load_database_url is None:
+        load_from_custom_url = (
+            Prompt.ask(
+                "Do you want to use a custom URL?",
+                choices=["Yes", "No"],
+                default="No",
+                console=console,
+            )
+            == "Yes"
+        )
+        if load_from_custom_url:
+            custom_load_database_url = Prompt.ask(
+                "Enter the URL to load the database with initial data from.",
+            )
+
+    if load_database:
+        url = (
+            custom_load_database_url
+            or "https://huggingface.co/datasets/cmu-lti/sotopia-pi/resolve/main/dump.rdb?download=true"
+        )
+        console.log(f"""Loading the database with initial data from {url}.
```

Comment:
So we will have many projects about sotopia, and I think people could choose which they wanna load? So I guess it's in general good to have different options here with corresponding citations. I added a bit more here

Reply:
Yes, they can install anything with custom url.

Reply:
I made some changes.

Reply:
add a menu in 2c046a1